webapp/app/forms/steps/lotse/personal_data.py

```diff
@@ -523,4 +523,29 @@ def render(self):
                                props=props_dict,
                                form=self.render_info.form,
                                header_title=_('form.lotse.header-title'))
-        
\ No newline at end of file
+        
+        
+class PersonAHasDisabilityPrecondition(BaseModel):
+    _step_to_redirect_to = StepPersonAHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+    
+    person_a_has_disability: str
+
+    @validator('person_a_has_disability', always=True)
+    def has_to_be_yes(cls, value):
+        if value != 'yes':
+            raise ValidationError
+        return value
+        
+class PersonBHasDisabilityPrecondition(BaseModel):
+    _step_to_redirect_to = StepPersonBHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+    
+    person_b_has_disability: str
+
+    @validator('person_b_has_disability', always=True)
+    def has_to_be_yes(cls, value):
+        if value != 'yes':
+            raise ValidationError
+        return value
+    
```

Comment:
I know this is in PR https://github.com/digitalservice4germany/steuerlotse/pull/516 but I need it here also.

Reply:
👍 When merging with main, we will need to make sure that this precondition is replaced with the correct one (in` has_disability.py`)

Reply:
I checked this.